OpenRA.Mods.Cnc/Traits/Disguise.cs

```diff
@@ -79,8 +79,8 @@ class DisguiseInfo : TraitInfo
 		[Desc("The condition to grant to self while disguised.")]
 		public readonly string DisguisedCondition = null;
 
-		[Desc("What diplomatic stances can this actor disguise as.")]
-		public readonly PlayerRelationship ValidStances = PlayerRelationship.Ally | PlayerRelationship.Neutral | PlayerRelationship.Enemy;
+		[Desc("Player relationships the owner of the disguise target needs.")]
+		public readonly PlayerRelationship ValidRelationships = PlayerRelationship.Ally | PlayerRelationship.Neutral | PlayerRelationship.Enemy;
```

Comment:
`ValidOwnerRelationships` / `ValidPlayerRelationships` I think would be better.
I can't think of any reasons to leave out the `Player` part when it is the most important (especially since docs aren't necessarily front-and-center when reading/writing MiniYaml).

Reply:
The value is a *Player*Relationship, which imo makes it clear that we're talking about players here. (I'm actually not sure which other relationships there might be?)

Reply:
I agree that the "*Relationships" naming may not be ideal, but i'm not immediately sure of a better suggestion. "ValidPlayers: Ally, Neutral" would read better IMO but might not be clear enough?

Reply:
> Is this better or worse?

IMO better, but I'd keep the generic wording: `ValidActorOwners` for the trait in question and others, and perhaps `AffectedActorOwners` or simply `AffectedPlayers` for remaining traits `CreatesShroud`, `InfiltrateForDecoration`, `VoiceAnnouncement`, `WithTextDecoration`, `WithRangeCircle`, `WithDecoration`, `WithBuildingRepairDecoration`, `AppearsOnRadar`, `TooltipDescription`, `RevealsShroud`. 

IMO the term "valid" is not really fitting for the second group because the event in question will happen anyway (provided any relation is defined), the question is only to whom it happens while for the first group of traits any event can only happen when the actor owner is in the right condition (indirectly true for `ProximityExternalCondition` and `GrantExternalConditionPower` since the condition is granted anyway, however the actor owner relation condition decides if any further event develops).

I don't know if introducing this separation is desirable though from the coder point of view.